config/routes.rb

```diff
@@ -172,6 +173,10 @@
           # See the above :admin_settings routes, outside of :communities resource
           get :settings,       to: redirect("/admin/settings")
           put :update_settings # PUT request, no redirect
+          get "getting_started_guide(/*all)", to: redirect {|p, req|
+            sub_path = p[:all] == "" ? "" : "/#{p[:all]}"
+            "#{p[:locale]}/admin/getting_started_guide#{sub_path}"
+          }
```

Comment:
What about just listing all the different routes here? That would be simple, super easy to understand, implicit and less magical. Yeah, a bit more repetition, but I'm not sure if that's so bad thing here.